src/kaltura-player.js

```diff
@@ -131,6 +131,7 @@ class KalturaPlayer extends FakeEventTarget {
     const playerConfig = Utils.Object.copyDeep(mediaConfig);
     Utils.Object.mergeDeep(playerConfig.sources, this._localPlayer.config.sources);
     Utils.Object.mergeDeep(playerConfig.session, this._localPlayer.config.session);
+    playerConfig.plugins = playerConfig.plugins || {};
```

Comment:
is this reflected in the `KPMediaConfig` object?

Reply:
should be change in provider types